src/apprunner/explorer/apprunnerServiceNode.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as AsyncLock from 'async-lock'
+import { AppRunnerClient } from '../../shared/clients/apprunnerClient'
+import { AppRunner } from 'aws-sdk'
+import { AppRunnerNode } from './apprunnerNode'
+import { CloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
+import { ext } from '../../shared/extensionGlobals'
+import { toArrayAsync, toMap } from '../../shared/utilities/collectionUtils'
+import { CloudWatchLogsParentNode } from '../../cloudWatchLogs/explorer/cloudWatchLogsNode'
+import { CloudWatchLogs } from 'aws-sdk'
+import { AWSResourceNode } from '../../shared/treeview/nodes/awsResourceNode'
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+const CONTEXT_BASE = 'awsAppRunnerServiceNode'
+
+const OPERATION_STATUS = {
+    START_DEPLOYMENT: localize('AWS.apprunner.operationStatus.deploy', 'Deploying...'),
+    CREATE_SERVICE: localize('AWS.apprunner.operationStatus.create', 'Creating...'),
+    PAUSE_SERVICE: localize('AWS.apprunner.operationStatus.pause', 'Pausing...'),
+    RESUME_SERVICE: localize('AWS.apprunner.operationStatus.resume', 'Resuming...'),
+    DELETE_SERVICE: localize('AWS.apprunner.operationStatus.delete', 'Deleting...'),
+    UPDATE_SERVICE: localize('AWS.apprunner.operationStatus.update', 'Updating...'),
+}
+
+type ServiceOperation = keyof typeof OPERATION_STATUS
+
+export class AppRunnerServiceNode extends CloudWatchLogsParentNode implements AWSResourceNode {
+    public readonly name: string
+    public readonly arn: string // TODO: fix package.json
```

Comment:
?

Reply:
Oh yeah, need to add the node for the copy ARN command.